SPECS/kmod-drbd.spec

```diff
@@ -21,6 +21,28 @@ BuildRequires: perl
 BuildRequires: %kernel_module_package_buildreqs
 %endif
 
+# XCP-ng patches
+#
+# Context
+# We use a modified version of DRBD due to changes added by linbit
+# that impact the mechanism for opening and closing DRBD resources.
+# On our side, the cost of changing our use of DRBD is significant
+# regarding the code base (risk of breaking a pool, blocking production, etc.),
+# however, it would be interesting to review our resource management
+# on SMAPIv3 in order to be compatible with the current linbit design.
+#
+# Patches generated from this repo/branch:
+# https://github.com/LINBIT/drbd/tree/restore_exact_open_counts_9.2.14
+# with this command: "git format-patch HEAD^^ --no-signature --no-numbered".
```

Comment:
Command only work for current branch, it would be better with tag, branch or SHA.